Renaming Use, Migrate, MST and troubleshooting (#4193)
timescale/docs: mst/user-management.md

# User management

You can add new users, and manage existing users, in $MST_CONSOLE_LONG. New users can be added to an entire project, or a single

> service.

Reviewer: ```suggestion
$MST_SERVICE_SHORT.
```

The `tsdbadmin` user is the owner of the database, but is not a superuser. To
access features requiring a superuser, log in as the `postgres` user instead.

> On $MST_LONG services, the `tsdbadmin` user can:

Reviewer: ```suggestion
The `tsdbadmin` user for $MST_LONGs can:
```

### Adding service users

> 1.  [Sign in][mst-login] to your $MST_CONSOLE_LONG. By

Reviewer: ```suggestion
1.  [Sign in][mst-login] to $MST_CONSOLE_LONG. By
```

## Multi-factor user authentication

> You can use multi-factor authentication (MFA) to log in to your $MST_LONG account. This requires an authentication code, provided by the

Reviewer: ```suggestion
You can use multi-factor authentication (MFA) to log in to $MST_CONSOLE_LONG. This requires an authentication code, provided by the
```

### Configuring multi-factor authentication

> 1.  [Sign in][mst-login] to your $MST_CONSOLE_LONG.

Reviewer: ```suggestion
1.  [Sign in][mst-login] to $MST_CONSOLE_LONG.
```

1.  Click the `User information` icon in the top-right of the dashboard to go to
    the `User profile` section.
1.  In the `Authentication` tab, toggle `Two-factor authentication` to
    `Enabled`, and enter your password.
1.  On your mobile device, open the Google Authenticator app, tap `+` and select
    `Scan a QR code`.
1.  On your mobile device, scan the QR code provided by $MST_LONG.
1.  In your $MST_SHORT dashboard, enter the confirmation
    code provided by the Google Authenticator app, and click
    `Enable Two-Factor Auth`.

</Procedure>

> <Highlight type="warning">

Reviewer: Add spaces after and before the metatags or the variables may not work
